docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -214,16 +214,18 @@ Scanning a repository from SCM providers such as GitHub or GitLab ensures that S
 1. Ensure you are signed in to Semgrep App.
 2. Click **Projects** on the left sidebar.
 3. Optional: If you do not see the repository you want to add in the **Projects** page of Semgrep app, follow the steps in the succeeding sections to ensure that Semgrep App can detect the repository.
-4. Click **Scan new project**, and then select which repository provider Semgrep to integrate with.
+4. Click **Scan new project**, and then click **Run scan in CI**.
+5. Select which repository provider Semgrep to integrate with.
 5. For **GitHub Actions**:
-    1. Click **Add CI Job **next to the name of the project to add.
-    2. Optional: If you do not see the repository you want to add, follow the succeeding guide on **detecting GitHub repositories**.
-    3. Click **Commit file**. This commits a `semgrep.yml` file containing pertinent scan parameters, such as schedule, what branch to scan on, and so on.
-    4. Semgrep App generates a token composed of a **Secret name** and **Secret value**. Copy and paste these into your repository's settings. Links are provided by Semgrep App.
-    5. Click **The secret's there, continue**.
-    6. Semgrep App creates a file for the CI job. Select toggles for desired features, such as scanning on a schedule.
+    1. Select the checkbox next to the name of the projects to add.
+    2. Optional: If you do not see the repository you want to add, follow the succeeding guide on **detecting GitHub repositories**. TODO

```

Comment:
@s-santillan the **detecting GitHub repositories** guide does not exist, right? Or was it somehow renamed? If not... I will create it, just asking because I was not sure. :-)

Reply:
It's in line 240, "to ensure that your GitHub repository is detected by Semgrep App"

Reply:
@s-santillan thank you! Adding as a subsection. 🙂 👯‍♂️